src/back/Update/HighPriority.php

```diff
@@ -123,6 +124,8 @@ public function update(): void
      */
     private function processSubsectionTopics(array $topics, callable $avgProcessor): void
     {
+        $config = $this->settings->get();
+
```

Comment:
For what?

Reply:
Sorry, the line is not needed. I thought I would take API minimum request threshold from the config, but there is another way how this setting is used.